jdbc/src/test/java/org/apache/zeppelin/jdbc/JDBCInterpreterTest.java

```diff
@@ -189,6 +189,20 @@ public void testSplitSqlQuery() throws SQLException, IOException {
   }
 
   @Test
+  public void testSqlQueryWithBackslash() throws SQLException, IOException {
+    String sqlQuery = "select '\\n', ';';" +
+        "select replace('A\\;B', '\\', 'text')";
```

Comment:
This `select replace('A\\;B', '\\', 'text')` looks incomplete, IMO it should be this `select replace('A\;B', '\'', 'text');`

Can you add one more line to test as well i.e. `select '\'', ';'`